tsconfig.base.json

```diff
@@ -17,6 +17,7 @@
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "resolveJsonModule": true,
+    "allowSyntheticDefaultImports": true,
```

Comment:
For what code do we need this?

Reply:
In order to mock platform for unit tests, I needed to create a spy for the `os.platform()` which required import directly from `os`. Unfortunately, this caused a default export TS error and I had to override it with this option.